coil-svg/src/commonMain/kotlin/coil3/svg/SvgDecoder.kt

```diff
@@ -27,21 +29,21 @@ import okio.use
  *
  * @param parser An [Svg.Parser] that converts bytes into an [Svg]. This property can be used
  *  to replace the default SVG parser with a different library.
+ * @param density A function whose return value will be used to scale the output dimensions of the
+ *  image if the target size is [Size.ORIGINAL].
  * @param useViewBoundsAsIntrinsicSize If true, uses the SVG's view bounds as the intrinsic size for
  *  the SVG. If false, uses the SVG's width/height as the intrinsic size for the SVG.
  * @param renderToBitmap If true, renders the SVG to a bitmap immediately after decoding. Else, the
  *  SVG will be rendered at draw time. Rendering at draw time is more memory efficient, but
  *  depending on the complexity of the SVG, can be slow.
- * @param scaleToDensity If true and the target size is [Size.ORIGINAL], multiplies the source
- *  dimensions of the image by the device's display density. NOTE: This only affects Android.
  */
 class SvgDecoder(
     private val source: ImageSource,
     private val options: Options,
     val parser: Svg.Parser = Svg.Parser.DEFAULT,
+    val density: PlatformContext.() -> Float = NO_DENSITY,
```

Comment:
We don't need to maintain binary compatibility here as the new constructor with `parser` hasn't been released yet.